docs/REST API/rest-api.md

```diff
@@ -3885,6 +3887,64 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Add labels to host
+
+Adds manual labels to a host. 
+
+`POST /api/v1/fleet/hosts/:id/labels`
+
+#### Parameters
+
+| Name | Type    | In   | Description                  |
+| ---- | ------- | ---- | ---------------------------- |
+| labels   | list | body | The list of label names to add to the host. |
+
+
+#### Example
+
+`POST /api/v1/fleet/hosts/12/labels`
+
+##### Request body
+
+```json
+{
+  "labels": ["label1", "label2"]
+}
+```
+
+##### Default response
+
+`Status: 200`
```

Comment:
@marko-lisica @lucasmrod do these endpoints respond with a 200 or 204? (I remember Martin mentioning we normally use 204 when there's no response data, e.g [here](https://fleetdm.com/docs/rest-api/rest-api#default-response54))

Reply:
HTTP 200. (Though HTTP 204 would make sense too.)